# Model comparison

Contrast the two recorded model scores and calculate the absolute macro-F1 gain.

**Synthetic evaluation context.** Helix Forge Lab evaluated Model Alpha M-14 on 2026-07-13 in run R184. The recorded macro F1 0.873 is paired with precision 0.881 and recall 0.865. Comparator Model Beta B-07 recorded macro F1 0.846. HF-Eval-S1/S2 v3.2 contains exactly 2,400 synthetic records (S1=1,600; S2=800) across five synthetic intent labels: account_access, delivery_status, plan_change, refund_request, and technical_help. No real subjects or live data are used.

This export is a self-contained offline synthetic analysis notebook.


In [1]:
alpha, beta = .873, .846
gain = alpha - beta
print(f'Model Alpha M-14: F1 {alpha:.3f}')
print(f'Model Beta B-07:  F1 {beta:.3f}')
print(f'absolute gain: {gain:.3f} ({gain * 100:.1f} points)')


Model Alpha M-14: F1 0.873
Model Beta B-07:  F1 0.846
absolute gain: 0.027 (2.7 points)


## Interpretation

Action note: report the 2.7-point Alpha advantage alongside the shared evaluation conditions.

The displayed values remain deterministic synthetic artifacts; no people, production systems, or network services were used.
